cvat/apps/engine/backup.py

```diff
@@ -21,11 +22,13 @@
 from zipfile import ZipFile
 
 import django_rq
+import json_stream
 from django.conf import settings
 from django.core.exceptions import ObjectDoesNotExist
 from django.db import transaction
 from django.utils import timezone
 from rest_framework import serializers, status
+from rest_framework.compat import SHORT_SEPARATORS
```

Comment:
I don't think `compat` is a public module. It would be better to refrain from using it.